Source/ACE.Entity/CharacterLevel.cs

```diff
@@ -6,7 +6,7 @@ namespace ACE.Entity
     public class CharacterLevel
     {
         [JsonProperty("level")]
-        public int Level { get; set; }
+        public uint Level { get; set; }
```

Comment:
Found that the character level was an integer and changed this to a uint to match data types across the source.